webapp/maps_heatmaps/test/geohash.spec.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as geokit from "geokit";
+import { expect } from "chai";
+import "mocha";
+
+describe("convert latlon to geohash function", () => {
+  it("correctly converts - standard coordinates", () => {
+    const coordinates = { lat: 70.2995, lng: -27.9993 };
+    const result = geokit.hash(coordinates, 12);
+    expect(result).to.equal("gkkpfves308e");
```

Comment:
with what did you compute these expected reference values?

Reply:
with python library geohash2 which we will use to store the geohash in the database.